---
title: 
output: html_document
css: "my1.css"
---

<style>
.main-container {
  max-width: 100% !important;
}
</style>


```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r klippy, echo=FALSE, include=TRUE}
klippy::klippy(position=c("top","right"))
```





<div class="part1">
<h1>SpaceX: Gene Co-Expression Network Estimation for
 Spatial Transcriptomics: Examples</h1>
 
<div class="tab">
<button class="tablinks button" onclick="location.href='htmlrmark1.html'">Home</button>
<button class="tablinks button" onClick="location.href='my2.html'">Installation</button> 
<button class="tablinks button" onclick="location.href='my3.html'">Examples</button>
<button class="tablinks button" onclick="location.href='my4.html'">Visualization</button>
<button class="tablinks button" onclick="location.href='my5.html'">Supplemental File</button>
<button class="tablinks button" onclick="location.href='my6.html'">Contact</button>
<button class="tablinks button" onclick="location.href='my7.html'">Citation</button>
<button class="tablinks button" onclick="location.href='my8.html'">View on Github</button>
<button class="tablinks button" onclick="location.href='my9.html'">Acknowledgement</button>
 </div>
  </div>




<h2>Consider the initial data.</h2>


```{r, eval=FALSE}
# Reading the Breast cancer data

# Spatial locations
head(BC_loc)

# Gene expression for data
head(BC_count) 

# Data processing
G <-dim(BC_count)[2] # number of genes
N <-dim(BC_count)[1] # number of locations 
```

Now, apply the SpaceX method on the Breast cancer dataset.

```{r, eval=FALSE}
# Application to SpaceX method
BC_fit <- SpaceX(BC_count,BC_loc[,1:2],BC_loc[,3],sPMM=FALSE,Post_process = TRUE)

# Shared_network :: Shared co-expression matrix
# Cluster_network :: Cluster specific co-expression matrices
```



